sqlmesh/dbt/model.py

```diff
@@ -346,9 +352,13 @@ def to_sqlmesh(self, context: DbtContext) -> Model:
             )
 
         if self.cluster_by:
-            optional_kwargs["clustered_by"] = [
-                d.parse_one(c, dialect=dialect).name for c in self.cluster_by
-            ]
+            clustered_by = []
+            for c in self.cluster_by:
+                try:
+                    clustered_by.append(d.parse_one(c, dialect=dialect).name)
+                except SqlglotError as e:
+                    raise ConfigError(f"Failed to parse cluster_by field '{c}': {e}") from e
```

Comment:
Small QOL - should we aggregate these errors so they're all reported at once after the loop instead of 1 by 1?

Reply:
Why? It seems like our general strategy has been failing fast